vault/changelog.release.2022-02-15.md

```diff
@@ -0,0 +1,77 @@
+---
+id: egxM8X43CqrQLuhYQ6Wu8
+title: '0.82'
+desc: ''
+updated: 1644904313283
+created: 1644902103166
+---
+
+Dendron 0.82 has sprouted  🌱
+
+Schema template variables can now include `CURRENT_WEEK`.
```

Comment:
this is a nice enhancement but wouldn't dedicate so much space to it. would update the announcement to something like below:

> Smaller changelog this week as the team is working on some bigger architectural changes to increase the general stability and performance of Dendron (our focus for this month). More updates on those in upcoming releases!